slipcover/__main__.py

```diff
@@ -3,16 +3,17 @@
 from typing import Any, Dict
 from slipcover import slipcover as sc
 from slipcover import bytecode as bc
+from slipcover import branch as br
+import ast
 import atexit
 
-
 from importlib.abc import MetaPathFinder, Loader
-from importlib.util import spec_from_loader
 
 class SlipcoverLoader(Loader):
-    def __init__(self, sci, orig_loader):
+    def __init__(self, sci, orig_loader, origin):
```

Comment:
Please add comments for each field in `self` as well as type annotations.

Reply:
Type annotations aren't present in `importlib.abc.Loader`...  I've guessed what I could, but left many thinking none are better than incorrect ones.